README.md

```diff
@@ -4,19 +4,64 @@
 [![GoDoc](https://godoc.org/github.com/istio/pilot?status.svg)](https://godoc.org/github.com/istio/pilot)
 [![codecov.io](https://codecov.io/github/istio/pilot/coverage.svg?branch=master)](https://codecov.io/github/istio/pilot?branch=master)
 
-Istio Pilot is the microservice proxy mesh orchestrator. It is responsible for dynamically
-configuring proxies in a cluster 
-platform environment to support L7-based routing, request destination policies (load balancing, circuit breaking), and point-to-point
-control policies such as fault injection, retries, and time-outs.
+Istio Pilot provides platform-independant service discovery, and exposes an
+interface to configure rich L7 routing features such as label based routing
+across multiple service versions, fault injection, timeouts, retries,
+circuit breakers. It translates these configurations into sidecar-specific
+configuration and dynamically reconfigures the sidecars in the service mesh
+data plane. Platform-specific eccentricities are abstracted and a
+simplified service discovery interface is presented to the sidecars based
+on the Envoy data plane API.
 
-Please see [istio.io](https://istio.io)
-to learn about the overall Istio project and how to get in touch with us. To learn how you can
-contribute to any of the Istio components, including Istio Pilot, please
-see the Istio [contribution guidelines](https://github.com/istio/istio/blob/master/CONTRIBUTING.md).
+Please see
+[Istio's traffic management concepts](https://istio.io/docs/concepts/traffic-management/overview.html)
+to learn more about the design of Pilot and the capabilities it provides.
 
-## Getting started
-
-Istio Pilot [design](doc/design.md) gives an architectural overview of its components - cluster platform abstractions, service model, and the 
+Istio Pilot [design](doc/design.md) gives an architectural overview of its
+internal components - cluster platform abstractions, service model, and the
 proxy controllers.
 
-If you are interested in contributing to the project, please take a look at the [build instructions](doc/build.md) and the [testing infrastructure](doc/testing.md).
+
+To learn how you can contribute to Istio Pilot, please
+see the Istio
+[contribution guidelines](https://github.com/istio/istio/blob/master/CONTRIBUTING.md).
+
+# Quick start
+
+1. *Install Bazel:* [Bazel 0.6.0](https://github.com/bazelbuild/bazel/releases/tag/0.6.0) or
```

Comment:
0.6.1 is the latest (some bug fixes). You can leave 0.6.0 here, I'm hoping 0.6.x series is somewhat backwards compatible.